### PR TITLE
Edit accepted data types

### PR DESCRIPTION
**Pull request by apete12**
What I did: Edit data types accepted by API to align with form inputs. Change port to 3001

What we learned: 

- The backend needs to be setup to accept whatever data types the form is sending it, and vice versa. Our form was built to send an object, but our backend wasn't equipped to accept that object. This was a new experience with debugging, because we originally thought it was our POST but our scope of debugging included the backend as well. Once we realized that, we looked to the backend and were able to debug

- Another learning curve is that the backend doesn't necessarily need to 'require' everything. if a pet owner is inputting details but the pet doesn't have pet facts or nicknames, we don't need to require it on the backend

- We also learned that small data can live in local storage instead of implementing a database, like PostgresSQL.

Testing Notes/Extras: 

Did this break anything?
- [X]  No
- [ ]  Yes

Type of change
- [ ]  New feature (non-breaking change which adds functionality)
- [ ]  Bug fix (non-breaking change which fixes an issue)
- [X]  Refactor(DRY-ing up/ reorganizing code, etc.)
- [ ]  Super small fix (Corrected a typo, removed a comment, etc.)
- [ ]  Added testing for: 
- [ ]  Skip all the other stuff and briefly explain the fix.

Checklist:
- [ ]  If this code needs to be tested, all tests are passing
- [X]  The code runs locally
- [X]  The code is DRY. If it's not, I tried my best
- [X]  I reviewed my code before pushing

What's next?
N/A

### CODE REVIEW

**Review by judy0ye — APPROVED** [review]
Thanks for changing all the properties to match!

**Review by paulina-isabel — APPROVED** [review]
Good catch!

File: server.js [hunk 1]
```diff
@@ -48,9 +56,9 @@ app.post('/api/v1/pets', (request, response) => {
     }
   }
 
-  const { name, type } = pet;
-  app.locals.pets.push({ name, type, id });
-  response.status(201).json({ name, type, id });
+  const { petName, petNickname, petAge, petFunFact, petOwnersName} = pet;
```
Comment (nicktu12) [inline server.js:59]: Naming things can be hard! Is it necessary to include the word `pet` in all these property values, or could you get away with `name, nickname, age, funFact, ownersName` - the inclusion of `pet` could be seen as duplicated code/naming

File: node_modules/.cache/@babel/register/.babel.7.22.17.development.json
Comment (nicktu12) [inline node_modules/.cache/@babel/register/.babel.7.22.17.development.json:1]: node modules can be added to your .gitignore - these files will be downloaded locally when another dev runs npm install

**Pull request merged**